### PR TITLE
Eligibility Report

### PR DESCRIPTION
**Pull request by cesar82**

### CODE REVIEW

File: src/components/pages/ReportsPage/EligibleScholarships/index.js [hunk 1]
```diff
@@ -1,62 +1,73 @@
...
+  const [to, setTo] = useState(new Date(moment(params?.to).toDate()))
 
-  const clientCell = ({length}, client) => (
-    <td rowSpan={length}>{client}</td>
-  )
+  useEffect(() => {
+    if (from && to) {
+      // setParams({ from: moment(from).format(DATE_DATA_FORMAT), to: moment(to).format(DATE_DATA_FORMAT) })
+    }
+  }, [from, to])
```
Comment (normandanielmq) [inline src/components/pages/ReportsPage/EligibleScholarships/index.js:18]: Is this on TODO?

File: src/components/pages/ReportsPage/EligibleScholarships/index.js [hunk 2]
```diff
@@ -1,62 +1,73 @@
...
-              <td>{count}</td>
-            </tr>
-          );
-        })
-    );
-  };
+    return data?.map(
+      ({ organizationName, programName, count}, index) =>
+        (
+          <tr key={index}>
```
Comment (normandanielmq) [inline src/components/pages/ReportsPage/EligibleScholarships/index.js:24]: Is there a programId available for key?

File: src/components/pages/ReportsPage/EligibleScholarships/index.js (hunk 1 above)
Comment (cesar82) [reply src/components/pages/ReportsPage/EligibleScholarships/index.js:18]: yep

**Review by fabian202 — APPROVED** [review]

File: src/components/pages/ReportsPage/EligibleScholarships/index.js (hunk 2 above)
Comment (normandanielmq) [reply src/components/pages/ReportsPage/EligibleScholarships/index.js:18]: Why this was marked as resolved with no change or response?

File: src/components/pages/ReportsPage/EligibleScholarships/index.js [hunk 3]
```diff
@@ -1,62 +1,69 @@
...
+        <DatePicker
+          dateFormat={DATE_APP_FORMAT}
+          customInput={<Input autoComplete="off" />}
+          closeButton
+          selected={from}
+          endDate={to}
+          onChange={onFromDateChange}
+          showMonthDropdown
+          showYearDropdown
+          onChangeRaw={(e) => e.preventDefault()}
```
Comment (normandanielmq) [inline src/components/pages/ReportsPage/EligibleScholarships/index.js:49]: `useCallback`

File: src/components/pages/ReportsPage/EligibleScholarships/index.js [hunk 4]
```diff
@@ -1,62 +1,69 @@
...
+          customInput={<Input autoComplete="off" />}
+          closeButton
+          disabled={!from}
+          selected={to}
+          onChange={onToDateChange}
+
+          minDate={from}
+          showMonthDropdown
+          showYearDropdown
+          onChangeRaw={(e) => e.preventDefault()}
```
Comment (normandanielmq) [inline src/components/pages/ReportsPage/EligibleScholarships/index.js:64]: useCallback

File: src/hooks/useReports.js [hunk 5]
```diff
@@ -31,11 +35,15 @@ export const useReports = (reportType) => {
...
     setCustomData,
+    getReportData,
+    reportType
   }), [
     data,
     loading,
     error,
     params,
     customData,
+    setCustomData,
```
Comment (normandanielmq) [inline src/hooks/useReports.js:46]: No need to listen to methods

**Pull request merged**